<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGBA::MemorySearch</class>
 <widget class="QWidget" name="QGBA::MemorySearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>813</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>540</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Memory Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTableWidget" name="results">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Address</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Current Value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="value"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QRadioButton" name="typeNum">
       <property name="text">
        <string>Numeric</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">type</string>
       </attribute>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="typeStr">
       <property name="text">
        <string>Text</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">type</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="bitsGuess">
       <property name="text">
        <string>Guess</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">width</string>
       </attribute>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QRadioButton" name="bits8">
       <property name="text">
        <string>1 Byte (8-bit)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">width</string>
       </attribute>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QRadioButton" name="bits16">
       <property name="text">
        <string>2 Bytes (16-bit)</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">width</string>
       </attribute>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QRadioButton" name="bits32">
       <property name="text">
        <string>4 Bytes (32-bit)</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">width</string>
       </attribute>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Number type</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QRadioButton" name="numGuess">
       <property name="text">
        <string>Guess</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QRadioButton" name="numDec">
       <property name="text">
        <string>Decimal</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QRadioButton" name="numHex">
       <property name="text">
        <string>Hexadecimal</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0" colspan="2">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Search type</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QRadioButton" name="opEqual">
       <property name="text">
        <string>Equal to value</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QRadioButton" name="opGreater">
       <property name="text">
        <string>Greater than value</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QRadioButton" name="opLess">
       <property name="text">
        <string>Less than value</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QRadioButton" name="opUnknown">
       <property name="text">
        <string>Unknown/changed</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QRadioButton" name="opDelta">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Changed by value</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="QRadioButton" name="opDelta0">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Unchanged</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QRadioButton" name="opDeltaPositive">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Increased</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QRadioButton" name="opDeltaNegative">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Decreased</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">op</string>
       </attribute>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="searchROM">
       <property name="text">
        <string>Search ROM</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="search">
       <property name="text">
        <string>New Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchWithin">
       <property name="text">
        <string>Search Within</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewMem">
       <property name="text">
        <string>Open in Memory Viewer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refresh">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QGBA::MemorySearch</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>opDelta0</sender>
   <signal>toggled(bool)</signal>
   <receiver>value</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>231</x>
     <y>768</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="type"/>
  <buttongroup name="op"/>
  <buttongroup name="width"/>
 </buttongroups>
</ui>
